injected/integration-test/page-objects/duckplayer-overlays.js

```diff
@@ -255,15 +265,12 @@ export class DuckplayerOverlays {
 
     /**
      * @param {keyof userValues} setting
-     * @return {Promise<void>}
      */
-    async userSettingIs(setting) {
```

Comment:
Did this have to change? It makes the PR much bigger.

Reply:
yes it did, because removing the duplication had a slight timing change. didn't seem too bad because it's purely test code that's changed (eg: zero production lines were touched)

Reply:
If you make the signature still a promise then it should reduce the overhead though right? I asked another question about ordering.

Reply:
Then we either need to eslint-ignore it (lacking `await`) or fake a promise return value. Since the ordering has changed anyway I don't think this will alter the line-count if that's what you are concerned about